core/local/steps/initial_diff.js

```diff
@@ -200,3 +211,15 @@ function debounce (waiting /*: WaitingItem[] */, events /*: AtomWatcherEvent[] *
     }
   }
 }
+
+function foundUntouchedFile (event, was) {
+  if (was && event.kind === 'file') {
+    const { ctime, mtime } = event.stats
+    const eventUpdateTime = Math.max(ctime.getTime(), mtime.getTime())
```

Comment:
Shouldn't we check `mtime` only regarding content?

Reply:
From what I understand, it may be safer actually, since some third-party code could modify content then revert mtime (but not ctime), but still improve performances.